cmd/audiusd/main.go

```diff
@@ -314,40 +315,35 @@ func connectGET[Req any, Res any](
 	return func(c echo.Context) error {
 		queryParams := c.QueryParams()
 		maxParams := 50
-		count := 0
-		paramMap := make(map[string]interface{})
-		for k, v := range queryParams {
-			if count >= maxParams {
-				return c.JSON(http.StatusBadRequest, map[string]string{"error": "too many query parameters"})
-			}
-			if len(v) == 1 {
-				val := v[0]
-				// Try int64
-				if i, err := strconv.ParseInt(val, 10, 64); err == nil {
-					paramMap[k] = i
-				} else if f, err := strconv.ParseFloat(val, 64); err == nil {
-					paramMap[k] = f
-				} else if b, err := strconv.ParseBool(val); err == nil {
-					paramMap[k] = b
-				} else {
-					paramMap[k] = val
-				}
-			} else {
-				paramMap[k] = v
-			}
-			count++
+		
+		if len(queryParams) > maxParams {
```

Comment:
in the vein of knowledge share, we do this thing w/ fiber on api, and it's really nice
https://github.com/AudiusProject/api/blob/main/api/v1_coins_members.go#L13

not sure how hard adding something like that is here